pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
     return size > threshold
 
 
+def _is_stripped(tensor):
+    if len(tensor.external_data) == 0:
+        return False
+    external_value = None
+    for external_data in tensor.external_data:
+        if external_data.key == 'location':
+            external_value = external_data.value
```

Comment:
`location` won't appear twice? (correct me if I'm wrong)
```suggestion
            external_value = external_data.value
            break
```

Reply:
Before code could not consider multiple external data, so fixed to handle them.